depression-and-anxiety/getting-therapy.md

```diff
@@ -72,3 +72,24 @@ They can help you with things like talking to other people about your problem or
 - [Mind](http://www.mind.org.uk/information-support/types-of-mental-health-problems/depression/)
 - [Rethink mental illness](http://www.rethink.org/diagnosis-treatment/conditions/depression)
 - [Big White Wall](https://www.bigwhitewall.com/landing-pages/landingv3.aspx)
+
+##You can get therapy online
+
+A type of therapy called cognitive behavioural therapy (CBT) is available online and it's may be free on the NHS, depending where you live. 
+
+The benefits of computerised CBT include:
+
+-  no waiting time
+-  you don't a lot of experience on computers to use them
+-  research shows they work 
+-  you can use them any time
+
+Your GP can refer you for computerised CBT or you can refer yourself at: 
+
+[Big White Wall](http://www.mind.org.uk/information-support/drugs-and-treatments/talking-treatments/types-of-therapy/#.VmBFKHbhCM8)
+
+[Buddy](https://www.buddyapp.co.uk/undergoing-therapy)
+
+[Fearfighter](http://www.fearfighter.com/)
+
+[Leso](http://iesohealth.com/accessing-ieso/patients/)
```

Comment:
Doh.  I was testing you.  Can you change?


On 3 December 2015 at 17:08, Dominic Baggott <notifications@github.com>
wrote:

> In depression-and-anxiety/getting-therapy.md
> <https://github.com/nhsalpha/alphahealth/pull/49#discussion_r46580015>:
>
> > +The benefits of computerised CBT include:
> > +
> > +-  no waiting time
> > +-  you don't a lot of experience on computers to use them
> > +-  research shows they work
> > +-  you can use them any time
> > +
> > +Your GP can refer you for computerised CBT or you can refer yourself at:
> > +
> > +[Big White Wall](http://www.mind.org.uk/information-support/drugs-and-treatments/talking-treatments/types-of-therapy/#.VmBFKHbhCM8)
> > +
> > +[Buddy](https://www.buddyapp.co.uk/undergoing-therapy)
> > +
> > +[Fearfighter](http://www.fearfighter.com/)
> > +
> > +[Leso](http://iesohealth.com/accessing-ieso/patients/)
>
> Should these four links be in a list, rather than separate paragraphs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nhsalpha/alphahealth/pull/49/files#r46580015>.
>